src/reader.cpp

```diff
@@ -1284,6 +1288,10 @@ void reader_data_t::fill_history_pager(bool new_search, history_search_direction
                 shared_this->history_pager_history_index_start = index;
                 shared_this->history_pager_history_index_end = result.final_index;
             }
+            shared_this->pager.extra_progress_text =
+                (result.can_go_back || result.can_go_further)
+                    ? L"Use Ctrl+R/Ctrl+S to show older/newer results"
```

Comment:
The obvious problem with this is that it won't see rebound keys.

Maybe "trigger search again for more results"? Maybe mention that these keys are the default?

Reply:
yeah I think it's fairly unconventional to mention the key names, let's not do that
I just wasn't sure how to mention Ctrl+S as well but it's not that important.